code/datums/traits/negative.dm

```diff
@@ -100,94 +100,337 @@
 	if(DT_PROB(0.05, delta_time))
 		SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "depression", /datum/mood_event/depression)
 
+
 /datum/quirk/family_heirloom
 	name = "Family Heirloom"
 	desc = "You are the current owner of an heirloom, passed down for generations. You have to keep it safe!"
-	value = -1
+	value = -2
 	mood_quirk = TRUE
 	var/obj/item/heirloom
 	var/where
 
 /datum/quirk/family_heirloom/on_spawn()
 	var/mob/living/carbon/human/H = quirk_holder
 	var/obj/item/heirloom_type
+	var/list/heirloom_table = list()
+	var/civ = FALSE
+	var/med = FALSE
```

Comment:
Why use these when you can just insert these into the code where the jobs are directly?

Reply:
I didn't want to put the same item to all people in the same department, like (scalpel,drapes,cautery,etc)*8.